src/linux/Packaging.Linux/build.sh

```diff
@@ -59,22 +62,28 @@ if [ -z "$VERSION" ]; then
     die "--version was not set"
 fi
 
-ARCH="`dpkg-architecture -q DEB_HOST_ARCH`"
-if test -z "$ARCH"; then
-  die "Could not determine host architecture!"
+if ! "$INSTALL_FROM_SOURCE"; then
```

Comment:
I _think_ `if ! <string>` is a Bash-ism. It would probably be better to use `if [ -z "$INSTALL_FROM_SOURCE" ]` here, as well as in similar other instances

Reply:
I unfortunately was not able to use `if [ -z "$INSTALL_FROM_SOURCE" ]`, since we're passing in true or false (which we need to use for our csproj config to work correctly). So, I instead switched to `if [ "$INSTALL_FROM_SOURCE" = false ]` which I believe is ok for shell scripts, but please do correct me if I've misunderstood. I was able to update `if ! $is_ci;` to `if [ -z $is_ci ];` in `install-from-source.sh` though, which is nice.

Reply:
Nice!